components/image-controller/production/stone-prd-rh01/resources/konflux-ci-pr-cronjob.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: redhat-appstudio-tekton-catalog-pr-cronjob
+  name: konflux-ci-pr-cronjob
```

Comment:
`pr` is ambiguous. Better use pull-request or merge-request.

Reply:
Renamed to `konflux-ci-pull-request-builds-pruner`